dist/redhat/scylla.spec

```diff
@@ -62,6 +62,14 @@ This package installs all required packages for ScyllaDB,  including
 install_arg="--housekeeping"
 %endif
 ./install.sh --packaging --root "$RPM_BUILD_ROOT" --p11-trust-paths /etc/pki/ca-trust/source:/usr/share/pki/ca-trust-source $install_arg
+%if "%{_sbindir}" == "%{_bindir}"
+# according to the latest FHS, fedora 42 unifies /usr/bin and /usr/sbin, and
+# its /usr/sbin/ becomes a symlink to bin. see
+# https://fedoraproject.org/wiki/Releases/42/ChangeSet#Unify_/usr/bin_and_/usr/sbin
+mv -v %{buildroot}/usr/sbin/scylla* %{buildroot}%{_bindir}/
```

Comment:
please note, i am using `${buildroot}` instead of `$RPM_BUILD_ROOT`. the idea is to follow the guidelines at https://docs.fedoraproject.org/en-US/packaging-guidelines/ , where `%{buildroot}` is recommended.